backend/src/main/java/atlantbh/restaurants/controllers/LocationController.java

```diff
@@ -0,0 +1,23 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.Location;
+import atlantbh.restaurants.services.LocationService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LocationController extends BaseController<Location, LocationService> {
+
+    public ResponseEntity allCities(@PathVariable String country) {
+        return ResponseEntity.ok(service.findAllCitiesForCountry(country));
+    }
+    public ResponseEntity allCountries() {
+        return ResponseEntity.ok(service.findAllCountires());
+    }
+  //  public ResponseEntity allCities() {
```

Comment:
Always remove commented code from PR. If you will need it future, leave TODO comment above (and then leave commented code).

Reply:
Fixed in 92c4f7264b274d7da369717e0ed1478a93d30ee4